shapenet2kubric/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:18.04
+
+# --- Install system dep
+RUN apt-get update --fix-missing
+RUN apt-get -y install cmake
+
+# --- install obj2gltf (REQUIRED)
+RUN apt install -y nodejs
+RUN apt install -y npm
+RUN npm install -g npm@6.9.0
+RUN npm install -g obj2gltf
+
+# --- Blender2.83 uses the system python3.7
```

Comment:
We don't need blender here at all do we?

Reply:
Yes, it's gone now, I only keep python3.7 in the latest commit.
Is that the best solution for it though?